helm/polaris/ci/fixtures/persistence.yaml

```diff
@@ -49,6 +49,8 @@ stringData:
           <property name="eclipselink.connection-pool.default.initial" value="1" />
           <property name="eclipselink.connection-pool.default.min" value="1" />
           <property name="eclipselink.connection-pool.default.max" value="1" />
+          <property name="eclipselink.session.customizer" value="org.apache.polaris.extension.persistence.impl.eclipselink.PolarisEclipseLinkSessionCustomizer" />
```

Comment:
How is this EclipseLink change related to the storage config locations limit?

Reply:
I think this is a merge conflict, let me yank it. Good catch!

Reply:
Yeah this is #1114, just a bad diff